lib/index.js

```diff
@@ -3,35 +3,40 @@
  *
  * Copyright (c) 2012-2017 Digital Bazaar, Inc. All rights reserved.
  */
-var async = require('async');
-var bedrock = require('bedrock');
-var brPermission = require('bedrock-permission');
-var crypto = require('crypto');
-var database = require('bedrock-mongodb');
-var util = require('util');
-var BedrockError = bedrock.util.BedrockError;
+const async = require('async');
+const bedrock = require('bedrock');
+const {config} = bedrock;
+const brPermission = require('bedrock-permission');
+const crypto = require('crypto');
+const database = require('bedrock-mongodb');
+const logger = require('./logger');
+const util = require('util');
+const BedrockError = bedrock.util.BedrockError;
 
 // load config defaults
 require('./config');
 
+let cache;
+const {enable: cacheEnabled} = config.key.cache;
+if(cacheEnabled) {
+  cache = require('bedrock-redis');
```

Comment:
since bedrock-redis is in "optionalDependencies" perhaps this should be in a try/catch